Assets/Scripts/Game/Leveling/AlignmentSystem.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.Events;
+
+namespace RPG
+{
+    public class AlignmentSystem : MonoBehaviour
+    {
+        public UnityEvent<int> onAligneChanged;
```

Comment:
Should probably be called `OnAlignmentChanged`. Also if you make this a c# property you can make it so that it's read-only, so people can't overwrite the event outside of this class:
```suggestion
        public UnityEvent<int> OnAlignmentChanged { get; private set; }
```